vertical-pod-autoscaler/deploy/admission-controller-deployment.yaml

```diff
@@ -43,6 +43,21 @@ spec:
             - containerPort: 8000
             - name: prometheus
               containerPort: 8944
+          livenessProbe:
+            httpGet:
+              path: /health-check
+              port: prometheus
```

Comment:
The name of this port is weird, since it's mixed used. 
I don't know if that happens much though

Reply:
yeah, seemed safer since port is a flag in code.